rmgpy/rmg/model.py

```diff
@@ -418,8 +418,12 @@ def make_new_reaction(self, forward, check_existing=True, generate_thermo=True):
         """
 
         # Determine the proper species objects for all reactants and products
-        reactants = [self.make_new_species(reactant, generate_thermo=generate_thermo)[0] for reactant in forward.reactants]
-        products = [self.make_new_species(product, generate_thermo=generate_thermo)[0] for product in forward.products]
+        try:
+            reactants = [self.make_new_species(reactant, generate_thermo=generate_thermo)[0] for reactant in forward.reactants]
+            products = [self.make_new_species(product, generate_thermo=generate_thermo)[0] for product in forward.products]
+        except:
+            logging.error("Error when making species in reaction {forward:s} from {forward.family:s}")
```

Comment:
```suggestion
            logging.error(f"Error when making species in reaction {forward:s} from {forward.family:s}")
```

Shall I pretend I left this mistake in to check our code review process?

Reply:
Ah, I should have caught this! Sorry